src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java

```diff
@@ -235,13 +192,11 @@ public void testSedTestingEndDateDisplayedInSedDetails(final String sedEndDate)
 
     /**
      * Load listing details to verify listing was uploaded successfully.
-     *
-     * @param ed
-     *            - edition digits in CHPL ID
+     * @param edition - edition digits in CHPL ID
      */
     @Then("^I see that listing was uploaded successfully to CHPL "
             + "and listing details load as expected for uploaded \"([^\"]*)\" listing$")
-    public void testVerifyUploadWasSuccessful(final String ed) {
+    public void testVerifyUploadWasSuccessful() {
```

Comment:
The step passes in a variable. If the method doesn't need the variable then the step should be updated to not pass it in